src/sentry/models/monitorcheckin.py

```diff
@@ -16,12 +16,22 @@
 
 class CheckInStatus:
     UNKNOWN = 0
+    """No status was passed"""
```

Comment:
Feels like these should be above or inline, and just use `#`

Reply:
But this puts them into the doc comments
![image](https://user-images.githubusercontent.com/1421724/213821909-e5773066-54e7-4f5d-937e-e96e95483f6c.png)